wormhole-connect/src/routes/abstracts/routeAbstract.ts

```diff
@@ -128,6 +128,7 @@ export abstract class RouteAbstract {
    * These operations have to be implemented in subclasses.
    */
   public abstract getMinSendAmount(routeOptions: any): number;
+  public abstract getMaxSendAmount(): number | undefined;
```

Comment:
You could simplify this by always returning `number`, and defaulting to `Infinity` instead of `undefined`.

Reply:
changed